rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/charts/SwapLineChart.java

```diff
@@ -50,6 +50,24 @@ public String apply(String s) {
                 return s + " Mb";
             }
         });
+        setTooltipItemHandler(new Function<String, String>() {
+            @Override
+            public String apply(String s) {
+                int indexOfSpace = s.lastIndexOf(" ");
+                String firstHalf = s.substring(0, indexOfSpace);
+                String number = s.substring(indexOfSpace + 1);
+
+                number = MBeanChart.keepNDigitsAfterComma(number, 0);
```

Comment:
Could you create different 'number' variables, if they are presenting different values?
So it will be no dependency on only 1 'number' variable.